changelog.txt

```diff
@@ -1,3 +1,17 @@
+Hector 2.0.0
+============
+* Incorporated 1-D diffusive ocean heat model (DOECLIM)
+* deleted the old temperature component
+* bugfix: double counting halocarbon radiative forcing
+* bugfix: re-enabled CO2 concentration constraint
+* initalized FSO2 first before FSO2i and FSO2d
+* getComponentName now public
```

Comment:
Why? Was this to better support pyhector? Maybe clarify

Reply:
I wouldn't bother with stuff like this in the change log, as it's not user-visible.  The change log is for telling users what has changed from their perspective.  What is different in the interface?  How will the results be different?  It isn't necessary or desirable to note implementation changes with no effects that are visible to users.

Reply:
yes. most of those changes after bug fix were to support pyhector

Reply:
You could collapse all such changes into a single line something like: 

* Various changes to internals to support pyhector

Reply:
Then maybe a single line saying "Variety of internal changes to better support calling Hector from external code like pyhector"

Reply:
Ha! Beat you by 18 seconds!